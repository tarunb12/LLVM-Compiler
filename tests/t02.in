// operation/printf test

int main() {
    printf("%d * (%d + %d) = %d\n", 5, 6, 7, 5 * (6 + 7))
    printf("(%d * %d) + %d = %d\n", 5, 6, 7, (5 * 6) + 7)
    printf("%d * %d + %d = %d\n", 5, 6, 7, 5 * 6 + 7)
    printf("%s %s\n", "string", "specifiers")
    printf("%c%c", '0' + '?', '4' + '7')
    // '0' = 48, '?' = 63, '0' + '?' = 111 = 'o'
    // '4' = 52, '7' = 55, '4' + '7' = 107 = 'k'
    return 0
}