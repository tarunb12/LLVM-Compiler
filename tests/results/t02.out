5 * (6 + 7) = 65
(5 * 6) + 7 = 37
5 * 6 + 7 = 37
string specifiers
ok